### ISSUE TITLE
Add the favicon 

### ISSUE THREAD

**Issue by mrinal41298** [garimasingh128/Hacktoberfest2020-MovieDo#13]
Create and add favicon for your website which will make it more spectacular.

**Comment by Halix267** [comment]
@garimasingh128 I want to work on this issue . Plzzz assign it to me

**Comment by Halix267** [comment]
@mrinal41298  I want to work on this issue .Plzz assign it to me.

**Comment by Vrindagupta6828** [comment]
@mrinal41298 please assign me this

**Comment by ShubhamSood1406** [comment]
@mrinal41298 Do you want the favicon next to the MovieGo title of the website?

**Comment by garimasingh128** [comment]
Yup. Just pick up a favicon which looks best acc to you in terms of design and simplicity and add it in index.html.

**Comment by An-prog-hub** [comment]
@mrinal41298 please assign me this to me

**Comment by Vrindagupta6828** [comment]
@mrinal41298 please can you tell me that only a single favicon can be added to a website so how can 2 people add favicon?
Me and halix267 as you have assigned to both of us

**Comment by garimasingh128** [comment]
Both of you can create a "Design" folder in the root of the repo and add your png images in that for the favicon. 
- No need to add it in the index.html for now
- We will only look for the best design
- Since its a design issue, you need to make a personalized favicon using canva, figma etc.

**Comment by Vrindagupta6828** [comment]
@garimasingh128 ok sure thanx

**Comment by Vrindagupta6828** [comment]
@garimasingh128  I have added the folder please review it.

**Comment by An-prog-hub** [comment]
@garimasingh128 sure

**Comment by Vrindagupta6828** [comment]
@garimasingh128 please label my pull request as hacktoberfest accepted #57
